doc/requirements-doc.txt

```diff
@@ -59,6 +59,7 @@ sphinxcontrib.yt==0.2.2
 sphinx-sitemap==2.2.0
 sphinx-thebe==0.1.1
 autodoc_pydantic==1.6.1
+sphinxcontrib-redoc==1.6.0
```

Comment:
I don't have a particular reason to pin the version here, just following what's been done for other sphinx extensions